src/annotate/celltypist/config.vsh.yaml

```diff
@@ -0,0 +1,114 @@
+  authors:
+    - __merge__: /src/authors/jakub_majercik.yaml
+      roles: [ author ]
+
+  argument_groups:
+    - name: Inputs
+      description: Arguments related to the input (aka query) dataset.
+      arguments:
+        - name: "--input"
+          alternatives: [-i]
+          type: file
+          description: Input h5mu file.
+          direction: input
+          required: true
+          example: input.h5mu
+        - name: "--modality"
+          description: Which modality to process.
+          type: string
+          default: "rna"
+          required: false
+        - name: "--input_obs_label"
+          type: string
+          description: Key in obs field of input adata for label information.
+          required: false
+    - name: Reference
+      description: Arguments related to the reference dataset.
+      arguments:
+        - name: "--reference"
+          type: file
+          description: "User-provided reference dataset. The data that will be used as reference to call cell types."
+          example: reference.h5ad
+          direction: input
+          required: false
+        - name: "--reference_obs_label"
+          type: string
+          description: Key in obs field of reference AnnData with cell-type information.
+          default: "cell_ontology_class"
+          required: false
+        - name: "--check_expression"
+          type: boolean
+          description: "Whether to check the expression of the reference dataset to the format reccomended by CellTypist."
+          default: false
+    - name: Outputs
+      description: Output arguments.
+      arguments:
+        - name: "--output"
+          type: file
+          description: Output h5mu file.
+          direction: output
+          example: output.h5mu
+        - name: "--output_compression"
+          type: string
+          choices: ["gzip", "lzf"]
+          required: false
+          example: "gzip"
+    - name: Model arguments
+      description: Model arguments.
+      arguments:
+        - name: "--model"
+          type: file
+          description: "Pretrained model in pkl format. If not provided, the model will be trained."
+          required: false
+          example: pretrained_model.pkl
+        - name: "--model_save_path"
+          type: file
+          direction: output
+          description: "Path to save the trained model."
+          required: false
+          example: trained_model.pkl
+        - name: "--feature_selection"
+          type: boolean
+          description: "Whether to perform feature selection."
+          default: true
+        - name: "--majority_voting"
+          type: boolean
+          description: "Whether to refine the predicted labels by running the majority voting classifier after over-clustering."
+          default: true
+        - name: "--only_train"
+          type: boolean
+          description: "Whether to only train the model without performing annotation."
+          default: false
```

Comment:
no use case for this atm, the goal would be to actually annotate cells using OP
```suggestion
```

Reply:
this option is there only to generate test model for now, should i remove it anyway?